doc/usage/bfcli.rst

```diff
@@ -101,9 +101,14 @@ Rules are defined such as:
 With:
   - ``$MATCHER``: zero or more matchers. Matchers are defined later.
   - ``counter``: optional literal. If set, the filter will counter the number of packets and bytes matched by the rule.
-  - ``$VERDICT``: action taken by the rule if the packet is matched against **all** the criteria: either ``ACCEPT`` or ``DROP``.
+  - ``$VERDICT``: action taken by the rule if the packet is matched against **all** the criteria: either ``ACCEPT``, ``DROP`` or ``CONTINUE``.
+    - ``ACCEPT``: Forward the packet to the kernel
+    - ``DROP``: Discard the packet.
+    - ``CONTINUE``: Continue processing subsequent rules.
```

Comment:
It would be worth updating the "Chains" section of the doc to mention that even though `CONTINUE` exists, it's not valid as a chain policy.